runtime/jcl/common/jclcinit.c

```diff
@@ -712,6 +716,38 @@ initializeRequiredClasses(J9VMThread *vmThread, char* dllName)
 		}
 	}
 
+	if (J9_ARE_ANY_BITS_SET(vm->extendedRuntimeFlags2, J9_EXTENDED_RUNTIME2_LOAD_AGENT_MODULE)
+			&& (NULL != vm->modulesPathEntry->extraInfo)) {
+		Trc_JCL_initializeRequiredClasses_addAgentModuleEntry(vmThread);
```

Comment:
An TraceEntry tracepoint needs to have a corresponding TraceExit tracepoint. I expect this one can just be declared as an TraceEvent since it doesn't correspond to function entry/exit.